_sass/_layout.scss

```diff
@@ -40,6 +40,32 @@ h4 {
   font-size: 24px;
 }
 
+.scroll-to-top {
+  display: none;
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  box-shadow: 0px 0px 10px $grey-color;
+  border: 2px solid $grayscale-dark;
+  font-size: 30px;
+  text-align: center;
+  color: $grayscale-dark;
+  background-color: #fff;
+  -webkit-transition: background 0.3s ease-in-out;
+  -moz-transition: background 0.3s ease-in-out;
+  transition: background 0.3s ease-in-out;
+  position: fixed;
+  bottom: 50px;
+  right: 10%;
```

Comment:
Let's fix this to 50px as well. The % doesn't scale very well in different screen sizes and sometimes it's a little too far inside.